app/uk/gov/hmrc/crdlcache/repositories/CustomsOfficeListsRepository.scala

```diff
@@ -121,7 +121,7 @@ class CustomsOfficeListsRepository @Inject() (val mongoComponent: MongoComponent
         previousInstruction.flatMap { _ =>
           nextInstruction match {
             case UpsertCustomsOffice(customsOffice) =>
-              logger.info(s"UpsertingCustomsOffice ${customsOffice.referenceNumber}")
+              logger.debug(s"UpsertingCustomsOffice ${customsOffice.referenceNumber}")
```

Comment:
Maybe we should take a similar approach to these logs i.e.
`Upserting customs office ${customsOffice.referenceNumber}` etc.